volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -1070,14 +1070,28 @@ def path(self) -> str:
             not current_dentry.is_root()
             and current_dentry.vol.offset not in dentry_seen
         ):
-            parent = current_dentry.d_parent
-            reversed_path.append(current_dentry.d_name.name_as_str())
+            try:
+                parent = current_dentry.d_parent
+            except exceptions.InvalidAddressException:
```

Comment:
These  statements end in the same result, they should probably be in a single try/except block?  The traceback would show which line went on, so this just seems excessive?